.circleci/config.yml

```diff
@@ -221,44 +180,35 @@ jobs:
       - store_artifacts:
           path: /go/out/istio-sidecar.deb
 
-  shellcheck:
-    docker:
-      - image: koalaman/shellcheck-alpine:v0.6.0
-    steps:
-      - checkout
-      - run:
-          name: Check Scripts
-          command: /root/project/bin/check_shell_scripts.sh
-
-  # This job is triggered via API. expect env vars
-  # DOCKER_USER, DOCKER_PASS, HUB, TAG
-  dockerpush:
-    <<: *defaults
-    resource_class: xlarge
-    steps:
-      - checkout
-      - run: make sync
-      - run: make build
-      - setup_remote_docker:
-          docker_layer_caching: true
-      - run: make docker.all
-      - run: docker login -u ${DOCKER_USER} -p ${DOCKER_PASS}
-      - run: make push
-
 workflows:
   version: 2
   all:
     jobs:
-      - shellcheck
-      - build
+      # The cloudfoundry test is not yet migrated to prow yet
+      - e2e-pilot-cloudfoundry-v1alpha3-v2
```

Comment:
this wont work without build.

Reply:
it seems to work fine, you can click on the job and see it runs